src/test/java/org/gridsuite/study/server/NetworkModificationTest.java

```diff
@@ -1675,6 +1675,61 @@ public void testLinesAttachToSplitLines() {
         verifyNetworkModificationPut(stubPutId, badBody);
     }
 
+    @SneakyThrows
+    @Test
+    public void testGeneratorScaling() {
+        String userId = "userId";
+        StudyEntity studyEntity = insertDummyStudy(UUID.fromString(NETWORK_UUID_STRING), CASE_UUID, "UCTE");
+        UUID studyNameUserIdUuid = studyEntity.getId();
+        UUID rootNodeUuid = getRootNode(studyNameUserIdUuid).getId();
+        NetworkModificationNode modificationNode = createNetworkModificationNode(studyNameUserIdUuid, rootNodeUuid, VARIANT_ID, "node", "userId");
+        UUID modificationNodeUuid = modificationNode.getId();
+
+        String generatorScalingRequest = "{\"type\":\"GENERATOR_SCALING\",\"variationType\":\"DELTA_P\",\"isIterative\":true,\"generatorScalingVariations\":[{\"filters\":[{\"id\":\"b5c93a96-b325-47a6-8303-c6764ba680e5\",\"name\":\"filter1\"},{\"id\":\"6c2137ec-6f2c-4aeb-ab76-e898263e972a\",\"name\":\"filter2\"}],\"variationValue\":\"300\",\"variationMode\":\"PROPORTIONAL_TO_PMAX\"}]}";
```

Comment:
here, the request content is not very important
so, to avoid having json string hard coded, use ModificationInfos.builder() and Objectmapper writeValueAsString method to generate the json request ...